data/projects/create_jenkins_plugin.md

```diff
@@ -6,20 +6,35 @@ If you find this fun you'll be the one knowing this component best and will
 get the chance to maintain it on your own if you want - always with the
 support of the community guarding your back of course.
 
+#### Motivation
+
+Jenkins is the most popular ci-cd server, and there are many corporations 
+who already use it. Integrating coala into Jenkins can attract business 
+clients better who could end up sponsoring/helping out coala.
```

Comment:
It also generally allows a wider range of users to use coala in their prefered way.